packages/overlay/stories/overlay-story-components.ts

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    LitElement,
+    html,
+    css,
+    property,
+    TemplateResult,
+    CSSResult,
+    CSSResultArray,
+} from 'lit-element';
+
+import { Placement } from '../';
+import { Radio } from '../../radio';
+import { Overlay } from '../';
+
+// Prevent infinite recursion in browser
+const MAX_DEPTH = 7;
+
+class OverlayTargetIcon extends LitElement {
+    static get styles(): CSSResult {
+        return css`
+            :host {
+                position: absolute;
+                display: block;
+                color: var(--spectrum-global-color-magenta-700);
+                width: 64px;
+                height: 64px;
+            }
+        `;
+    }
+
+    public render(): TemplateResult {
+        return html`
+            <svg
+                aria-hidden="true"
+                focusable="false"
+                data-prefix="fas"
+                data-icon="bullseye"
+                class="svg-inline--fa fa-bullseye fa-w-16"
+                role="img"
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 496 512"
+            >
+                <path
+                    fill="currentColor"
+                    d="M248 8C111.03 8 0 119.03 0 256s111.03 248 248 248 248-111.03 248-248S384.97 8 248 8zm0 432c-101.69 0-184-82.29-184-184 0-101.69 82.29-184 184-184 101.69 0 184 82.29 184 184 0 101.69-82.29 184-184 184zm0-312c-70.69 0-128 57.31-128 128s57.31 128 128 128 128-57.31 128-128-57.31-128-128-128zm0 192c-35.29 0-64-28.71-64-64s28.71-64 64-64 64 28.71 64 64-28.71 64-64 64z"
+                ></path>
+            </svg>
+        `;
+    }
+}
+customElements.define('overlay-target-icon', OverlayTargetIcon);
+
+class OverlayDrag extends LitElement {
```

Comment:
Nit: Since we have `hover` and `click` content, would it make sense for this to follow without having to mouse down?

Reply:
Hmm, I started to implement that, but it strikes me that it would be harder to test mouse hover if the target sticks to the mouse. It gets hard to un-hover. So, it is awkward either way. I'm open to suggestions if you have ideas on how to make the interaction better...